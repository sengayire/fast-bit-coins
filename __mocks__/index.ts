jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: (str) => str,
      i18n: {
        changeLanguage: () => new Promise(() => {})
      }
    }
  },
  initReactI18next: {
    type: '3rdParty',
    init: () => {}
  }
}))

jest.mock('react-redux', () => ({
  useSelector: jest.fn(),
  useDispatch: () => jest.fn()
}))
