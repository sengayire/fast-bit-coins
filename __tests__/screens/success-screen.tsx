import { NavigationContainer } from '@react-navigation/native'
import { render } from '@testing-library/react-native'

import '../../__mocks__'
import SuccessScreen from 'src/screens/verification-success-screen'

describe('Success screen', () => {
  it('should render  Success screen', () => {
    const view = render(
      <NavigationContainer>
        <SuccessScreen />
      </NavigationContainer>
    )

    expect(view).toMatchSnapshot()
  })
})
