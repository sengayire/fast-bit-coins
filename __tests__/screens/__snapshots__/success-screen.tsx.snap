// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success screen should render  Success screen 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      {
        "padding": 32,
      }
    }
  />
  <Text
    style={
      {
        "color": "#101820",
        "fontSize": 30,
        "fontWeight": "700",
        "paddingBottom": 24,
      }
    }
  >
    email.created
  </Text>
  <Text
    style={
      {
        "color": "#707070",
        "fontSize": 21,
        "lineHeight": 32,
      }
    }
  >
    email.text
  </Text>
</View>
`;
